### PR TITLE
Cloud9 Support for Layered Stacks

### PR DESCRIPTION
**Pull request by AvinashSeelam9**
## What does this PR do and why?
This PR is to provide Cloud9 Support for Layered Stacks
_Describe in detail what your merge request does and why._
This PR is to provide Cloud9 Support for Layered Stacks
## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/intermediate/templates/layered-stacks/index.md [hunk 1]
```diff
@@ -70,17 +70,34 @@ Outputs:
...
-1. Navigate through the wizard leaving everything default.
-1. On the Review page, scroll down to the bottom and click on **Create stack**.
-
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+  1. In the **Cloud9 terminal** navigate to `cfn101-workshop/code/workspace/layered-stacks`.
+  1. **Create Stack** by using the following AWS CLI command. The template requires you provide the values for `AvailabilityZones` parameter, For example `us-east-1a` and `us-east-1b` are used below. Please select 2 Availability Zone based on your region. 
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation create-stack \
+    --stack-name cfn-workshop-vpc \
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:79]:
```suggestion
    --stack-name cfn-workshop-layered-stacks-vpc \
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 2]
```diff
@@ -70,17 +70,34 @@ Outputs:
...
+    --template-body file://vpc.yaml \
+    --parameters ParameterKey=AvailabilityZones,ParameterValue=us-east-1a\\,us-east-1b
+  :::
+  1. Wait until the stack creation is completed by running the following AWS CLI command
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation wait stack-create-complete \
+    --stack-name cfn-workshop-vpc
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:86]:
```suggestion
    --stack-name cfn-workshop-layered-stacks-vpc
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 3]
```diff
@@ -70,17 +70,34 @@ Outputs:
...
+  :::
+  ::::
+	::::tab{id="local" label="Local development"}
+  1. Navigate to CloudFormation in the console and click **Create stack With new resources (standard)**.
+  1. In **Prepare template** select **Template is ready**.
+  1. In **Template source** select **Upload a template file**.
+  1. Choose a file `vpc.yaml`.
+  1. Enter a **stack name**. For example, `cfn-workshop-vpc`.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:94]:
```suggestion
  1. Enter a **stack name**. For example, `cfn-workshop-layered-stacks-vpc`.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 4]
```diff
@@ -70,17 +70,34 @@ Outputs:
...
+  1. For the **AvailabilityZones** parameter, select **2 AZs**.
+  1. You can leave the rest of the parameters **default**.
+  1. Navigate through the wizard leaving everything default.
+  1. On the Review page, scroll down to the bottom and click on **Create stack**.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:98]:
```suggestion
  1. On the Review page, scroll down to the bottom and click on **Submit**.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 5]
```diff
@@ -96,16 +113,32 @@ Outputs:
...
-1. Click **Next**.
-1. Navigate through the wizard leaving everything default.
-1. **Acknowledge IAM capabilities** and click on **Create stack**.
-
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+  1. Let's **Create Stack** by using the following AWS CLI command. The template requires you to specify `CAPABILITY_IAM` capability for creating IAM resources.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation create-stack \
+    --stack-name cfn-workshop-iam \
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:121]:
```suggestion
    --stack-name cfn-workshop-layered-stacks-iam \
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 6]
```diff
@@ -96,16 +113,32 @@ Outputs:
...
+    --template-body file://iam.yaml \
+    --capabilities CAPABILITY_IAM
+  :::
+  1. Wait until the stack creation is completed by running the following AWS CLI command
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation wait stack-create-complete \
+    --stack-name cfn-workshop-iam
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:128]:
```suggestion
    --stack-name cfn-workshop-layered-stacks-iam
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 7]
```diff
@@ -96,16 +113,32 @@ Outputs:
...
+  :::
+  ::::
+	::::tab{id="local" label="Local development"}
+  1. Navigate to CloudFormation in the console and click **Create stack With new resources (standard)**.
+  1. In **Prepare template** select **Template is ready**.
+  1. In **Template source** select **Upload a template file**.
+  1. Choose a file `iam.yaml`.
+  1. Enter a **stack name**. For example, `cfn-workshop-iam`.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:136]:
```suggestion
  1. Enter a **stack name**. For example, `cfn-workshop-layered-stacks-iam`.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 8]
```diff
@@ -96,16 +113,32 @@ Outputs:
...
+  1. Click **Next**.
+  1. Navigate through the wizard leaving everything default.
+  1. **Acknowledge IAM capabilities** and click on **Create stack**.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:139]:
```suggestion
  1. **Acknowledge IAM capabilities** and click on **Submit**.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 9]
```diff
@@ -180,16 +213,31 @@ WebServerSecurityGroup:
...
-1. You can leave the rest of the parameters **default**.
-1. Navigate through the wizard leaving everything default.
-1. On the **Review page**, scroll down to the bottom and click on **Create stack**.
-
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+  1. Let's **Create Stack** by using the following AWS CLI command. The template requires you to specify `CAPABILITY_IAM` capability for creating IAM resources.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation create-stack \
+    --stack-name cfn-workshop-ec2 \
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:221]:
```suggestion
    --stack-name cfn-workshop-layered-stacks-ec2 \
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 10]
```diff
@@ -180,16 +213,31 @@ WebServerSecurityGroup:
...
+    --template-body file://ec2.yaml 
+  :::
+  1. Wait until the stack creation is completed by running the following AWS CLI command
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation wait stack-create-complete \
+    --stack-name cfn-workshop-ec2
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:227]:
```suggestion
    --stack-name cfn-workshop-layered-stacks-ec2
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 11]
```diff
@@ -180,16 +213,31 @@ WebServerSecurityGroup:
...
+  :::
+  ::::
+	::::tab{id="local" label="Local development"}
+  1. Navigate to CloudFormation in the console and click **Create stack With new resources (standard)**.
+  1. In **Prepare template** select **Template is ready**.
+  1. In **Template source** select **Upload a template file**.
+  1. Choose a file `ec2.yaml`.
+  1. Enter a **stack name**. For example, `cfn-workshop-ec2`.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:235]:
```suggestion
  1. Enter a **stack name**. For example, `cfn-workshop-layered-stacks-ec2`.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 12]
```diff
@@ -180,16 +213,31 @@ WebServerSecurityGroup:
...
+  1. You can leave the rest of the parameters **default**.
+  1. Navigate through the wizard leaving everything default.
+  1. On the **Review page**, scroll down to the bottom and click on **Create stack**.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:238]:
```suggestion
  1. On the **Review page**, scroll down to the bottom and click on **Submit**.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 13]
```diff
@@ -212,13 +260,28 @@ If you not sure how to do that, follow the instructions from the [Session Manage
...
-1. In the pop-up window click on **Delete stack**.
-1. Hit the **refresh** button a few times until you see in the status **DELETE_COMPLETE**.
-1. Now you can delete **IAM** and **VPC** stack in any order as there are no more dependencies.
-
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+  1. **Delete Stack** by running the following AWS CLI command
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation delete-stack \
+    --stack-name cfn-workshop-ec2
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:268]:
```suggestion
    --stack-name cfn-workshop-layered-stacks-ec2
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 14]
```diff
@@ -212,13 +260,28 @@ If you not sure how to do that, follow the instructions from the [Session Manage
...
+  :::
+  1. Wait until the stack is deleted by using the following AWS CLI command.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation wait stack-delete-complete \
+    --stack-name cfn-workshop-ec2
+  :::
+  1. Repeat steps (1-2) above for stacks: `cfn-workshop-iam` and `cfn-workshop-vpc2`.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:275]:
```suggestion
  1. Repeat steps (1-2) above for stacks: `cfn-workshop-layered-stacks-iam` and `cfn-workshop-layered-stacks-vpc`.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 15]
```diff
@@ -212,13 +260,28 @@ If you not sure how to do that, follow the instructions from the [Session Manage
...
+  ::::
+	::::tab{id="local" label="Local development"}
+  1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the **EC2 stack**, for example `cfn-workshop-ec2`.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:278]:
```suggestion
  1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the **EC2 stack**, for example `cfn-workshop-layered-stacks-ec2`.
```

File: content/intermediate/templates/layered-stacks/index.md [hunk 16]
```diff
@@ -212,13 +260,28 @@ If you not sure how to do that, follow the instructions from the [Session Manage
...
+  1. In the top right corner, click on **Delete**.
+  1. In the pop-up window click on **Delete stack**.
```
Comment (sdangol) [inline content/intermediate/templates/layered-stacks/index.md:280]:
```suggestion
  1. In the pop-up window click on **Delete**.
```

**Review by sdangol — CHANGES_REQUESTED** [review]
Added a few suggestions on changes to workshop names

**Review by sdangol — APPROVED** [review]
LGTM

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**